shm/inc/item.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "cargo.hpp"
+
+class Item : public Cargo {
+public:
+    enum class Rarity { common = 1,
+                        rare = 2,
+                        epic = 4,
+                        legendary = 8 };

```

Comment:
Why explicit values and why powers of two?

Reply:
I think that multipliers for price counting should be precisely selected during "balancing" the game. That's why I've selected some temporary ones :D